include/udx.h

```diff
@@ -82,6 +82,13 @@ typedef struct udx_socket_send_s udx_socket_send_t;
 typedef struct udx_stream_write_s udx_stream_write_t;
 typedef struct udx_stream_send_s udx_stream_send_t;
 
+/* todo: should this be merged into the stream_write_t object? */
+typedef struct {
+  uv_buf_t buf;
+  udx_stream_write_t *write;
+  unsigned is_write_end : 1;
```

Comment:
```suggestion
  bool is_write_end;
```

C bitfields are really wonky and even at best the struct will be appropriately padded anyway.

Reply:
changed